src/participants/trigger_service.py

```diff
@@ -174,12 +177,28 @@ def handle_wi(self, wid):
             #self.obs.getCreatePackage(str(project), str(package))
         #else:
         try:
-            core.show_files_meta(self.obs.apiurl, str(project), str(package), expand=False, meta=True)
+            pkginfo=core.show_files_meta(self.obs.apiurl, str(project), str(package), expand=False, meta=True)
```

Comment:
I didn't actually implement 8d0bd4c on the server - which explains why I didn't catch it :(